jekyll/_cci2/configuration-reference.md

```diff
@@ -198,10 +200,10 @@ It's possible to specify multi-line `command`:
       make test
 ```
 
-Note that default `shell` has `-e` option which makes command to:
+Note that default `shell` has `-e` option which causes commands to:
```

Comment:
Something's off in this line. Should it be:

"Note that *by* default `shell` has *the* `-e` option which causes..."